proposals/0084-disable-rent-fees-collection.md

```diff
@@ -0,0 +1,64 @@
+---
+simd: '0084'
+title: Disable rent fees collection
+authors:
+  - Haoran Yi
+category: Standard
+type: Core
+status: Draft
+created: 2023-11-03
+feature: https://github.com/solana-labs/solana/issues/33946
+---
+
+## Summary
+
+Add a new feature to disable rent fees collections.
+
+## Motivation
+
+The total number of rent paying accounts on solana network is phasing out. And
+it is no longer possible to create new rent paying account on the network.
+Therefore, when all the rent paying accounts are gone in the network, we would
+like to disable rent fee collection on the network through a feature.
```

Comment:
The number of rent paying accounts is about 220K as of a few months ago and the last one will be drained of lamports in about one year.

Reply:
As of today, there are no more rent-paying accounts on MNB. All these rent paying accounts have already been topped off to be rent-exempt.